### ISSUE TITLE
not working in latest 3.4.3?

### ISSUE THREAD

**Issue by pixelman** [focuslabllc/craft-cheat-sheet#62]
I tried the basic setup but It returns a 'page not found'…
domain/cheatsheet/

**Comment by alsessions** [comment]
I've run into this if I try to use it in production. When the environment is set to "dev" it works as expected -- I assume this is intentional.

**Comment by bfordham** [comment]
Yes, it looks to be intentional. See https://github.com/focuslabllc/craft-cheat-sheet/blob/master/src/controllers/RoutesController.php#L30
